config.json

```diff
@@ -634,6 +634,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 9
+      },
+      {
+        "slug": "flatten-array",
+        "name": "Flatten Array",
+        "uuid": "39b6cd35-ab3c-4041-ad53-c1e405aa890d",
+        "difficulty": 3,
```

Comment:
This track only uses three difficulty values. Please change to "2" and sort exercise by difficulty.

Reply:
[e0d087b](https://github.com/exercism/jq/pull/176/commits/e0d087b2f1c7941612a09229ce0839c4b50931fe) aims to address this.